src/components/Chips/Chips.tsx

```diff
@@ -11,7 +11,7 @@ import { getTestId } from "../../tests/test-ids-utils";
 import { ChipsSize } from "./ChipsConstants";
```

Comment:
is useHover part of index and publishedcomponents files? if not, let's add it to there (otherwise it will not be expose)

Reply:
Thanks for noticing!